lib/fake_braintree/sinatra_app.rb

```diff
@@ -28,6 +32,52 @@ def hash_from_request_body_with_key(key)
       end
     end
 
+    # braintree.api.Client.prototype.tokenizeCard()
+    get '/merchants/:merchant_id/client_api/v1/payment_methods/credit_cards' do
+      request_hash = params
+
+      callback = request_hash.delete('callback')
+      nonce = FakeBraintree::PaymentMethod.tokenize_card(request_hash['creditCard'])
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.